src/utils/url.ts

```diff
@@ -39,6 +42,21 @@ export const openRoomUrl = (url: string) => {
 
 export const enterVenue = (venueId: string) => openUrl(venueInsideUrl(venueId));
 
+export const joinVenue = (
+  venueId?: string,
+  venueEntrance?: EntranceStepConfig[]
+) => {
+  if (!venueId) return;
+
+  const hasVenueEntrance = venueEntrance ? !!venueEntrance.length : false;
```

Comment:
Do we want to pass the whole `venueEntrance` in here, just to check if it is empty? Or should we make the param a boolean instead, and push that logic outside of this helper function?

Reply:
Moved it outside as a prop.